include/rmm/mr/device/detail/arena.hpp

```diff
@@ -344,6 +366,33 @@ class global_arena final {
     }
   }
 
+  /**
+   * @brief Dump memory to log.
+   *
+   * @param logger the spdlog logger to use
+   */
+  void dump_memory_log(std::shared_ptr<spdlog::logger> const& logger) const
+  {
+    lock_guard lock(mtx_);
+
+    logger->info("  Maximum size: {}", human_size(maximum_size_));
```

Comment:
These would change like so:
```suggestion
    logger->info("  Maximum size: {}", bytes{maximum_size_});
```